src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ApplicationSummary.vue

```diff
@@ -144,18 +144,18 @@
     <div v-if="currentStep !== 3">
       <v-card elevation="0" rounded="0" class="border-t border-b">
         <v-card-text>
-          <p>We'll review your application after we receive all references and documents.</p>
+          <p>We will review your application after we receive all references and documents.</p>
```

Comment:
Was this content changed in Figma to match? It will need to be for QA to stop the related stories coming back with discrepancies.

Reply:
I did a quick search for the above snippet and they are all changed in Figma.